kentikapi/client.go

```diff
@@ -31,6 +38,13 @@ type Client struct {
 	Alerting *resources.AlertingAPI
```

Comment:
// Batch comment above is redundant. Please remove it

Reply:
What was the meaning of "// Batch" comment before?

Reply:
Good question. This is a placeholder for Batch API, which apparently is not implemented. It is implemented in Python library tough. I created a ticket to fill that gap: https://clkentik.atlassian.net/browse/KNTK-343

FYI @mmac-m3a

Reply:
Done